.github/workflows/notify-module-version.yml

```
@@ -0,0 +1,29 @@
+name: Notify MODULE_VERSION changed
+
+on:
+  push:
+    branches:
+    - master
+    - release-*
+    paths:
+    - kernel-modules/MODULE_VERSION
+
+jobs:
+  notify:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Slack notification
+      uses: rtCamp/action-slack-notify@v2
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_COLLECTOR_NOTIFICATIONS_WEBHOOK }}
+        SLACK_CHANNEL: oncall
+        SLACK_COLOR: ${{ job.status }}
+        SLACK_LINK_NAMES: true
+        SLACK_TITLE: MODULE_VERSION has changed!
+        MSG_MINIMAL: commit
+        SLACK_MESSAGE: |
+          @collector-team you should now update downstream with the new version and commit.
+          Create a MR to <https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/tree/rhacs-0.1-rhel-8|rhacs-0.1-rhel-8>
+          adding this `MODULE_VERSION` to `upstream_sources.yml` and `distgit/containers/rhacs-drivers-build/container.yaml.in`.
+          Once done, you will need to create another MR to <https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/tree/rhacs-1.0-rhel-8|rhacs-1.0-rhel-8>
+          updating the support package used in `distgit/containers/rhacs-collector/pre-build-script`.
```

Let's turn this into a list for more clarity
```suggestion
         1.  Create a MR to <https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/tree/rhacs-0.1-rhel-8|rhacs-0.1-rhel-8>
         adding this `MODULE_VERSION` to `upstream_sources.yml` and `distgit/containers/rhacs-drivers-build/container.yaml.in`.
         2. Once done, you will need to create another MR to <https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/tree/rhacs-1.0-rhel-8|rhacs-1.0-rhel-8>
         updating the support package used in `distgit/containers/rhacs-collector/pre-build-script`.
```